RecoTracker/FinalTrackSelectors/plugins/TrackTfClassifier.cc

```diff
@@ -0,0 +1,123 @@
+#include "RecoTracker/FinalTrackSelectors/interface/TrackMVAClassifier.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "getBestVertex.h"
+
+#include "TrackingTools/Records/interface/TfGraphRecord.h"
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+#include "RecoTracker/FinalTrackSelectors/interface/TfGraphDefWrapper.h"
+
+struct TfDnnCache {
+  TfDnnCache() : session_(nullptr) {}
+  tensorflow::Session* session_;
+};
+
+namespace {
+  struct tfDnn {
```

Comment:
coding rule 2.6 [1]:
>For class, struct, type, and enumeration names use upper class initials, e.g. GeometryBuilder.

[1] https://cms-sw.github.io/cms_coding_rules.html

Reply:
In addition, why struct and not class? Seems like this is a nontrivial object with state, initialization etc.